README.md

```diff
@@ -58,16 +76,15 @@ See [here](https://docs.ansible.com/ansible/devel/dev_guide/developing_collectio
 
 ### Communication
 
-We have a dedicated Working Group for Ansible development.
+We announce releases and important changes through the [Ansible Bullhorn newsletter](https://github.com/ansible/community/issues/546). Be sure you are subscribed.
```

Comment:
I'm not sure if this is necessary here, as that issue's notification, maybe too much for a user wanting to be notified about `c.g.` only.

Reply:
There are also info about core releases, other collections, and contributor summits. I'll add a note that "If you are a contributor blah blah"

Reply:
done, PTAL